tests/integ/scala/test_dataframe_suite.py

```diff
@@ -910,6 +913,7 @@ def test_sample_on_union(session):
     )
 
 
+@pytest.mark.localtest
```

Comment:
I know this is from the existing code but why are we only calling `DataFrame.show()` without asserting anything. Let's improve this by asserting captured stdout like https://docs.pytest.org/en/7.1.x/how-to/capture-stdout-stderr.html#accessing-captured-output-from-a-test-function

Reply:
updated